json-server/routes.json

```diff
@@ -1,3 +1,4 @@
 {
-    "/api/v1/*": "/$1" 
-} 
\ No newline at end of file
+    "/api/v1/*": "/$1",
+    "/api/v1/deputies/:id": "/deputies/:id"
```

Comment:
This line shouldn't be necessary. The first one has a wildcard that will pass through anything after `/api/v1`. You only need to add custom routes in here where you need the behaviour to change.